Gopkg.toml

```diff
@@ -16,7 +16,7 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "SAAS-237-Check-Results-API"
```

Comment:
change it back to PMM-2.0 once https://github.com/percona/pmm/pull/560 is merged